website/docs/docs/deploy/continuous-integration.md

```diff
@@ -34,6 +34,7 @@ The [dbt Cloud scheduler](/docs/deploy/job-scheduler) executes CI jobs different
 - **Concurrent CI checks** &mdash; CI runs triggered by the same dbt Cloud CI job execute concurrently (in parallel), when appropriate.
 - **Smart cancellation of stale builds** &mdash; Automatically cancels stale, in-flight CI runs when there are new commits to the PR.
 - **Run slot treatment** &mdash; CI runs don't consume a run slot.
+- **SQL linting**<Lifecycle status="beta" /> &mdash; When enabled, automatically lints all SQL files in your project.
```

Comment:
Add: as a run step before your CI job builds. I.e. we do this first like a pre-commit because if you fail linting (and you have things not set to continue) then you don't have to waste compute on trying to build a PR that didn't yet fit what you are looking for in terms of code quality.